.github/workflows/add-to-project.yml

```diff
@@ -5,13 +5,23 @@ on:
     types:
       - opened
       - transferred
+env:
```

Comment:
The same comment here about the vars from the other PR.

Reply:
Completed, I had left it thinking we might use it in the future. We should be all set now.